docs/quick-starts/framework/next-auth/README.mdx

```diff
@@ -0,0 +1,42 @@
+---
+slug: /quick-starts/next-auth
```

Comment:
Do we have a logo for this guide? Maybe you can use `logoFilename: 'next.svg',` as a `sidebar_custom_props`

Reply:
added